src/jit/codegenxarch.cpp

```diff
@@ -5948,6 +6007,21 @@ void CodeGen::genCallInstruction(GenTreePtr node)
                         inst_RV_RV(ins_Copy(regType), allocatedReg, returnReg, regType);
                     }
                 }
+
+#ifdef FEATURE_SIMD 
+                // A Vector3 return value is stored in xmm0 and xmm1. 
+                // RyuJIT assumes that the upper unused bits of xmm1 are cleared but
+                // the native compiler doesn't guarantee it.
+                if (returnType == TYP_SIMD12) 
+                {
+                    regNumber retReg = returnReg = retTypeDesc->GetABIReturnReg(1);
```

Comment:
Why assignment to retunReg?

It is not required.  It is just some local for use within else {} block.